frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx

```diff
@@ -32,8 +35,8 @@ export const GlobalStyles = (): JSX.Element => {
       `,
     )}
 
-    ${aceEditorStyles}
-    ${saveDomImageStyles}
+      ${aceEditorStyles}
```

Comment:
let me know if your prettier config isn't formatting this - I'll need to check my IDE config if that's the case